src/types/types.js

```diff
@@ -231,7 +231,8 @@ export type BuySellPlugin = {
   cryptoCodes?: Array<string>,
   countryCodes?: { [string]: boolean },
   forPlatform?: string,
-  forCountries?: Array<string>
+  forCountries?: Array<string>,
+  addOnUrl?: string
```

Comment:
If you want to document `addOnUrl`, this would be the place.

Reply:
Done.